### PR TITLE
Add Manitoba shields

### LINKED ISSUE #229: Manitoba highway shields
Similar to Alberta:

- Trans-Canada Highway routes (Highways 1 and 16, network=CA:transcanada): Maple leaf (PR at #226)
- Provincial trunk highways (numbers 1-199, save Highways 1 and 16, network=CA:MB:PTH): squared-off shield (see also #218)
- Provincial roads (numbers 200-632, network=CA:MB:PR): oblong.
- Winnipeg city routes (network=CA:MB:Winnipeg): trapezoid (see example: [Route 105 shield](https://commons.m.wikimedia.org/wiki/File:Winnipeg_city_route_105.svg))

Similar with Alberta (see #218), the regular Manitoba provincial highway marker contains the shield within a rectangle with the provincial wordmark on top (see [this shield for Highway 10](https://commons.m.wikimedia.org/wiki/File:Manitoba_Highway_10.svg)), but the shield itself only (also used in interchange guide signs) has been preferred for simplicity, and also in line with practice by most commercial road maps (GMaps, most print maps). The official Manitoba highway map (see https://www.gov.mb.ca/mit/map/pdf/mapside.pdf ) also does the same, but uses a maple leaf completely filled with green for TCH routes.

### PR DESCRIPTION
**Pull request by claysmalley**
Fixes #229.

Screenshot from 2022-04-07 17-52-03
Screenshot from 2022-04-07 17-51-17

For Provincial Trunk Highways (`network=CA:MB:PTH`), the text "Manitoba" is removed. Letter suffixes are given the same font size as the rest of the ref, and a wider 3-character variant is introduced.

Manitoba_Highway_15
Manitoba_Highway_83A

For Provincial Roads, (`network=CA:MB:PR`), the text "Manitoba" is removed as well as the border, leaving just the black oval.

Manitoba_secondary_207

For Winnipeg city routes (`network=CA:MB:Winnipeg`), the text "Route" is removed and the colo(u)rs are adjusted to match the existing palette.

Winnipeg_city_route_115

### CODE REVIEW

File: style/icons/shield40_ca_mb_winnipeg.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg width="26" height="20" xmlns="http://www.w3.org/2000/svg">
+ <path d="m 0.5,0.5 0.6315789,3 H 24.868421 L 25.5,0.5 Z" fill="#fff" stroke="#000" style="fill:#006747;fill-opacity:1;stroke:#006747;stroke-opacity:1"/>
```
Comment (Pengor) [inline style/icons/shield40_ca_mb_winnipeg.svg:2]:
```suggestion
 <path d="m 0.5,0.5 0.6315789,3 H 24.868421 L 25.5,0.5 Z" fill="#006747" stroke="#006747"/>
```
Looks like there's some unused style info here that can go away

File: style/icons/shield40_ca_mb_winnipeg.svg [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+ <path d="M 1.1315789,3.5 4.5,19.5 h 17 l 3.368421,-16 z" fill="#fff" stroke="#000"/>
+ <path style="fill:none;fill-rule:evenodd;stroke:#000;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1" d="m 25.5,0.5 v 0"/>
```
Comment (Pengor) [inline style/icons/shield40_ca_mb_winnipeg.svg:4]:
```suggestion
```
It doesn't look like this path actually draws anything and can be removed

**Review by Pengor — CHANGES_REQUESTED** [review]
One thing I might add is rounded corners at the top two nodes of the green trapezoid and bottom two nodes of the black trapezoid

**Comment by ZeLonewolf** [comment]
Test and looks good from a visual perspective.

**Comment by claysmalley** [comment]
Rounded the corners of the Winnipeg shield, and removed unnecessary SVG elements and attributes.

Screenshot from 2022-04-08 15-52-13
Screenshot from 2022-04-08 15-51-45

**Review by ZeLonewolf — APPROVED** [review]

**Review by 1ec5 — APPROVED** [review]

**Comment by claysmalley** [comment]
Were we waiting on anything before merging this?

**Comment by ZeLonewolf** [comment]
I resolved conflicts but didn't have the chance to confirm I didn't break anything.

**Pull request merged**